src/app/(admin)/admin/_components/ServiceProvider/Organization/OrganizationTypesSelect.js

```diff
@@ -0,0 +1,32 @@
+import PropTypes from 'prop-types';
+import { AutocompleteArrayInput, ReferenceArrayInput, SelectArrayInput, useGetList } from 'react-admin';
+import { RESOURCES } from '../../../_lib/consts';
+
+export function OrganizationTypesSelect({ label, type = 'create', ...props }) {
+  const { data: typesList, isLoading: typesLoading } = useGetList(RESOURCES.organizationType);
+
+  return (
+    <>
+      {type === 'create' && (
```

Comment:
move it to consts
```py
types = { create: 'create, edit: 'edit' }
...
OrganizationTypesSelect({ label, type = types.create, ...props }) {
...
{type === types.create && (
...
OrganizationTypesSelect.propTypes = {
  type: PropTypes.oneOf([Object.values(types)],
  ...
};

Reply:
Thanks for comment! Extracted form types into separate configuration
 `const FORM_TYPES = {
  create: 'create',
  edit: 'edit',
};`